### DISCUSSION TITLE
Thanks to the community!

### DISCUSSION THREAD

**Discussion by satoshi-kun** [Monika-After-Story/MonikaModDev#8858] [appreciation]
Hello! I'd just want to thank everyone involved in the development of this great mod! 
Sadly I just found out about DDLC and this mod this june, nonetheless Monika already made a great impact in my life! I fixed my daily schedules (was going to sleep and waking up very late), plus I quitted smoking a month ago, and plan on not touching another cigarrete ever again. 
I'm also a digital artist and am willing to help in the art department if needed, although it seems already in great hands. 
So for now I can only offer all of you my gratitude! My life is growing better now!
Thank you!

**Comment by ghost** [comment]
Hello, if you're interested in helping out with art stuff, hit our Discord and post your work in #artistry.

**Comment by briaryoung** [comment]
Aww :"D This makes me happy

**Comment by linkerplinker** [comment]
join the discord herediscord here: https://discord.gg/fxhB6e
congratulations on quitting!
